src/NHibernate.Test/NHSpecificTest/GH1228/Fixture.cs

```diff
@@ -7,126 +7,95 @@ public class Fixture : BugTestCase
 		[Test]
 		public void TestOk()
 		{
-			using (ISession s = OpenSession())
+			using var s = OpenSession();
 			{
-				using (ITransaction t = s.BeginTransaction())
-				{
-					try
-					{
-						{
-							var queryThatWorks = s.CreateQuery(@"
+				var queryThatWorks = s.CreateQuery(
+					@"
 							SELECT ROOT FROM NHibernate.Test.NHSpecificTest.GH1228.Sheet AS ROOT 
 							WHERE (EXISTS (FROM NHibernate.Test.NHSpecificTest.GH1228.Shelf AS inv  
 										   , ROOT.Folder AS ROOT_Folder 
 										   WHERE ROOT_Folder.Shelf = inv AND inv.Id = 1
 								  )	   ) 
 								  AND ROOT.Name = 'SomeName'");
```

Comment:
Resulting indentation adjustment:

```suggestion
			var queryThatWorks = s.CreateQuery(
				@"
				SELECT ROOT FROM NHibernate.Test.NHSpecificTest.GH1228.Sheet AS ROOT
					WHERE (EXISTS (FROM NHibernate.Test.NHSpecificTest.GH1228.Shelf AS inv
									, ROOT.Folder AS ROOT_Folder
								WHERE ROOT_Folder.Shelf = inv AND inv.Id = 1
							))
						AND ROOT.Name = 'SomeName'");
```